src/firejail/profile.c

```diff
@@ -1494,7 +1494,7 @@ int profile_check_line(char *ptr, int lineno, const char *fname) {
 		else if (strncmp(ptr, "rlimit-fsize ", 13) == 0) {
 			cfg.rlimit_fsize = parse_arg_size(ptr + 13);
 			if ( cfg.rlimit_fsize == 0 ) {
-				perror("Error: invalid rlimit-fsize in profile file. use only non-negative numbers and k,m,g suffix for size");
+				perror("Error: invalid rlimit-fsize in profile file. only use positive numbers and k, m or g suffix.");
```

Comment:
It's a capital O (not o) after completing a sentence:
perror("Error: invalid rlimit-fsize in profile. Only use positive numbers and k, m or g suffix.");
Please change this and we're set for merge.